Makefile

```diff
@@ -63,15 +65,11 @@ plasma-server:
 		plasma_store -m 1000000 -s $(PLASMA_DB)
 
 run-wtr:
-		PKG_CONFIG_PATH=/opt/miniconda/lib/pkgconfig make
-		PKG_CONFIG_PATH=/opt/miniconda/lib/pkgconfig \
-			LD_LIBRARY_PATH=/opt/miniconda/lib \
-			go run ./_misc/wtr.go -db /tmp/plasma.db -id $(ID)
+		make
```

Comment:
What is this make command for?

Reply:
To build before running `wtr`, this diff is not great.